app/views/catalog/_show_actions_box_collection.html.erb

```diff
@@ -1 +1,40 @@
-<% # intentionally left blank to not render the actions box for collections %>
+<div class='al-show-actions-box col-md-3'>
+
+  <div class='al-collection-id'>
+    <% if document.unitid %>
+      <%= t(:'arclight.views.show.collection_id', id: document.unitid) %>
+    <% end %>
+
+    <% if document.unitid && document.containers.present? %>
+      <span> | </span>
+    <% end %>
+
+    <% if document.containers.present? %>
+      <span><%= document.containers.join(', ') %></span>
+    <% end %>
+  </div>
+
+    <% document_downloads = Arclight::DocumentDownloads.new(document) %>
+    <% if document_downloads.files.present? %>
+      <div class="mr-auto al-show-actions-box-downloads">
+
+        <% document_downloads.files.each do |file| %>
+          <div class="al-show-actions-box-downloads-file">
+            <% if file.pdf? %>
```

Comment:
Instead of the conditional checks here, could we do something like this?

```ruby
<%= blacklight_icon(file.type, classes: "al-show-actions-box-downloads-file") %>
```